ports/eathread/vcpkg.json

```
@@ -1,9 +1,10 @@
 {
   "name": "eathread",
   "version-string": "1.32.09",
-  "port-version": 2,
+  "port-version": 3,
   "description": "Electronic Arts Thread Library. EAThread implements a unified cross-platform interface for multithreaded programming on various platforms.",
   "homepage": "https://github.com/electronicarts/EAThread",
+  "supports": "!uwp & x64",
```

```suggestion
  "supports": "(!uwp | !arm) & x64",
```
I assume it should have `arm`?

---

x64 already implies !arm